src/_includes/components/board-member-item.liquid

```diff
@@ -1,11 +1,13 @@
 {% assign members = collections.boardMemberItem %}
 
 {% for member in members %}
-  <div class="font-serif flex flex-col items-center text-center px-4 pb-4 mx-auto desktop:pb-0 desktop:mt-4">
-    <img
-      src="{{ member.data.memberImage }}"
-      alt="{{ member.data.memberImageAlt }}"
-      class="w-full h-auto pb-6 desktop:pb-4 xl:w-[18.75rem] xl:h-[19.75rem]" />
+  <div class="font-serif flex flex-col items-center text-center px-4 pb-4 mx-auto dedesktop:pb-0 desktop:mt-4">
+    <div class="h-[119px] w-[95px] tablet:h-[234px] tablet:w-[210px] desktop:h-[324px] desktop:w-[300px]">
```

Comment:
Fine since we're just copying the old layout but generally, we'd want to define these in `em` and ideally, use the out of the box tailwind sizes when possible for the images, as well as, the tailwind breakpoints.